src/Server.cpp

```diff
@@ -648,7 +648,8 @@ void cServer::Shutdown(void)
 	// Notify the tick thread and wait for it to terminate:
 	m_TickThread.Stop();
 
-	cRoot::Get()->SaveAllChunks();
+	// Save all chunks in all worlds, wait for chunks to be saved:
```

Comment:
I just looked at cWorld::SaveAllChunks and apparently it only adds it to the world storage queue, it doesn't wait for it to empty. However cRoot waits for the thread to finish to it's still ok, just the comment might be wrong. Could you confirm?

Reply:
Your comment is completely correct, I'll update my comment.